test/sql.test.js

```diff
@@ -503,4 +577,68 @@ describe('sql connector', function() {
     expect(function() { runExecute(); }).to.not.throw();
     ds.connected = true;
   });
+
+  it('should throw if invalid order by statement is used by all', function(done) {
+    connector.all('customer', {order: 'n?ame'}, {}, (err) => {
+      expect(err).to.include({
+        message: 'Unknown property n?ame is used for model customer',
+        code: 'UNKNWON_PROPERTY',
+        statusCode: 400,
+        status: 400,
+      });
+      done();
+    });
+  });
+
+  it('should throw if invalid where statement is used by count', function(done) {
+    connector.count('customer', {'n?ame': 1}, {}, (err) => {
+      expect(err).to.include({
+        message: 'Unknown property n?ame is used for model customer',
+        code: 'UNKNWON_PROPERTY',
+        statusCode: 400,
+        status: 400,
+      });
+      done();
+    });
+  });
+
+  it('should throw if invalid where statement is used by update', function(done) {
+    connector.update('customer', {'n?ame': 1}, {name: 5}, {}, (err) => {
+      expect(err).to.include({
+        message: 'Unknown property n?ame is used for model customer',
+        code: 'UNKNWON_PROPERTY',
+        statusCode: 400,
+        status: 400,
+      });
+      done();
+    });
+  });
+
+  it('should throw if invalid where statement is used by destroyAll', function(done) {
+    connector.destroyAll('customer', {'n?ame': 1}, {}, (err) => {
+      expect(err).to.include({
+        message: 'Unknown property n?ame is used for model customer',
+        code: 'UNKNWON_PROPERTY',
+        statusCode: 400,
+        status: 400,
+      });
+      done();
+    });
+  });
+
+  it('should ignore if invalid order by statement is used by all', function(done) {
```

Comment:
If find it a bit difficult to spot why some tests are ignoring invalid statements while others are throwing an error. Can you please group the tests for "should ignore" using `describe` block?

```js
describe('with ignoreUnknownProperties: true', () => {
  it('should ignore if invalid order by statement is used by all', function(done) {
    // ...
  });
  // ...
});
```